frontend/libs/studio-components/src/components/StudioTableLocalPagination/StudioTableLocalPagination.stories.tsx

```diff
@@ -23,32 +24,34 @@ const meta: Meta = {
     emptyTableMessage: {
       description: 'The message to display when the table is empty.',
     },
-    isSortable: {
-      description:
-        'Boolean that sets sorting to true or false. If set to false, the sorting buttons are hidden.',
-    },
     pagination: {
       description:
         'An object containing pagination-related props. If not provided, pagination is hidden.',
     },
   },
 };
 
-export const Preview: Story = (args) => (
-  <StudioTableLocalPagination
-    columns={columns}
-    rows={rows}
-    size={args.size}
-    emptyTableMessage={'No data found'}
-    isSortable={true}
-    pagination={{
-      pageSizeOptions: [5, 10, 20, 50],
-      pageSizeLabel: 'Rows per page',
-      nextButtonText: 'Next',
-      previousButtonText: 'Previous',
-      itemLabel: (num) => `Page ${num}`,
-    }}
-  />
-);
+export const Preview: Story = (args) => {
+  const paginationTexts: PaginationTexts = {
+    pageSizeLabel: 'Rows per page:',
+    totalRowsText: 'Total number of rows:',
+    nextButtonAriaLabel: 'Next',
+    previousButtonAriaLabel: 'Previous',
+    numberButtonAriaLabel: (num) => `Page ${num}`,
+  };
+
+  return (
+    <StudioTableLocalPagination
+      columns={columns}
+      rows={rows}
+      size={args.size}
+      emptyTableFallback={'No data found'}
+      pagination={{
+        pageSizeOptions: [5, 10, 20, 50],
```

Comment:
Was 100 not an option?
Could we store this array somewhere more global?

Not so familiar with storybook and what is best practice there

Reply:
This is just an example implementation in Storybook, so that we can easily see how to use the component, and see which props it takes.

I think it is easier to see how to use the component if we leave it as is 😊 

![image](https://github.com/Altinn/altinn-studio/assets/148075168/8f617d6c-14ae-4eb0-9af8-eb782b9bd55b)

Reply:
Okey, sound good to me! :D But still include 100, or?